PanoptesNetClient/PanoptesNetClient/Auth.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Net;
+using System.Net.Http;
+
+namespace PanoptesNetClient
+{
+    static class Constants
+    {
+        public const int BearerTokenExpirationAllowance = 60 * 1000;
```

Comment:
Will this value be used outside of the Auth class? May just be better putting it into the Auth class

Reply:
Yes, I think this is a good approach. I initially placed it in a separate class as I was researching a few approaches, but it should probably be tied to the Auth class.